src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/metrics_data_source_profile/dummy_metrics_grid.tsx

```diff
@@ -0,0 +1,185 @@
+    });
+
+    return options;
+  }, [fields, breakdownField]);
+
+  const onChange = useCallback<NonNullable<ToolbarSelectorProps['onChange']>>(
+    (chosenOption) => {
+      const newField = chosenOption?.value
+        ? fields.find((currentField) => currentField.name === chosenOption.value)
+        : undefined;
+      setBreakdown(newField);
+    },
+    [fields, setBreakdown]
+  );
+
+  // The structure here is VERY similar to that of the `chart.tsx` component
+  // Perhaps there should be a shared template for this?
+  return (
+    <EuiFlexGroup
+      className="metricsExperience__chart"
+      direction="column"
+      alignItems="stretch"
+      gutterSize="none"
+      responsive={false}
+    >
+      <EuiFlexItem grow={false} css={chartToolbarCss}>
+        <EuiFlexGroup
+          direction="row"
+          gutterSize="s"
+          responsive={false}
+          alignItems="center"
+          justifyContent="spaceBetween"
+        >
+          <EuiFlexItem grow={false} css={{ minWidth: 0 }}>
+            <EuiFlexGroup direction="row" gutterSize="s" responsive={false} alignItems="center">
+              <EuiFlexItem grow={false}>
+                {renderToggleActions ? renderToggleActions() : null}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false} css={{ minWidth: 0 }}>
+                <div>
+                  {/* if we need to use <ToolbarSelector /> for our custom dropdowns, 
+                  this component needs to be moved to another package  */}
+                  <ToolbarSelector
+                    data-test-subj="metricsExperienceBreakdownSelector"
+                    data-selected-value=""
+                    searchable
+                    buttonLabel="Metrics Experience Breakdown by"
+                    popoverTitle={i18n.translate(
+                      'metricsExperience.breakdownFieldSelector.breakdownFieldPopoverTitle',
+                      {
+                        defaultMessage: 'Select breakdown field',
+                      }
+                    )}
+                    optionMatcher={comboBoxFieldOptionMatcher}
+                    options={fieldOptions}
+                    onChange={onChange}
+                  />
+                </div>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <IconButtonGroup
+              legend={i18n.translate('metricsExperience.chartActionsGroupLegend', {
+                defaultMessage: 'Chart actions',
+              })}
+              buttonSize="s"
+              buttons={actions}
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <section css={histogramCss} tabIndex={-1}>
+          {'Dummy Metrics Grid'}
+        </section>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+};
+
+// eslint-disable-next-line import/no-default-export
+export default DummyMetricsGrid;
```

Comment:
It can also live outside of Discover in a separate package or connected via https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/discover_shared/README.md if there are cross plugin dependencies.